README.md

```diff
@@ -105,8 +105,19 @@ Run unit tests with `make test`.
 
 #### Integration tests
 
-The continuous integration runs a set of [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features/evaluation.feature) using [`flagd`](https://github.com/open-feature/flagd).
-If you'd like to run them locally, you can start the flagd testbed with `docker run -p 8013:8013 ghcr.io/open-feature/flagd-testbed:latest` and then run `make integration-test`.
+The continuous integration runs a set of [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features) using [`flagd`](https://github.com/open-feature/flagd).
+If you'd like to run them locally, first pull the `test-harness` git submodule
+```
+git submodule update --init --recursive
+```
+then start the flagd testbed with 
+```
+docker run -p 8013:8013 -v $PWD/test-harness/testing-flags.json:/testing-flags.json ghcr.io/open-feature/flagd-testbed:latest
```

Comment:
Not bothered upgrading the version as it'll shortly be superseded by https://github.com/open-feature/go-sdk/pull/135